src/core/annotation.js

```diff
@@ -193,13 +243,9 @@ var Annotation = (function AnnotationClosure() {
 
     isInvisible: function Annotation_isInvisible() {
```

Comment:
Since I'm looking at this code, I couldn't help wondering if `isInvisible/isViewable/isPrintable` couldn't just be getters instead?
@timvandermeij If you agree with this, feel free to address that in either this, or a future, PR.